### PR TITLE
Docsp 28285- serializer allowed types

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-28285
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/csharp/docsworker-xlarge/docsp-28285-serializer-allowed-types/faq/#what-object-types-can-be-serialized-

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/fundamentals/data-formats/bson.txt [hunk 1]
```diff
@@ -55,8 +55,8 @@ example BSON document. Each key-value pair in the ``BsonDocument`` object is a
 
 .. literalinclude:: ../../includes/fundamentals/code-examples/Bson.cs
    :start-after: start-create
-   :end-before: newRestaurant.Add
-   :language: csharp 
+   :end-before: newRestaurant.Add(new BsonElement("restaurant_id", "12345"));
```
Comment (mongoKart) [inline source/fundamentals/data-formats/bson.txt:58]: fixing build error

File: source/usage-examples/deleteMany.txt [hunk 2]
```diff
@@ -63,8 +63,8 @@ code.
       :tabid: delete-many-sync
 
       .. literalinclude:: ../includes/code-examples/delete-many/DeleteMany.cs
-         :start-after: start-delete-many
-         :end-before: end-delete-many
+         :start-after: start-delete-many-builders
+         :end-before: end-delete-many-builders
```
Comment (mongoKart) [inline source/usage-examples/deleteMany.txt:67]: fixing build error

**Review by jordan-smith721 — APPROVED** [review]
LGTM with a couple suggestions

File: source/faq.txt [hunk 3]
```diff
@@ -264,4 +264,46 @@ following code demonstrates:
...
+.. _csharp-faq-object-serializer:
+
+What Object Types Can Be Serialized?
+------------------------------------
+
+The ``ObjectSerializer`` allows serialization and deserialization only of types
+that are considered safe. When you construct an ``ObjectSerializer``,
+you can pass in a delegate of type ``Func<Type, bool>`` that accepts 
+an object type and returns a boolean value indicating whether the
+type is safe for serialization.
```
[the remark this answers is not in the document]
Comment (mongoKart) [reply source/faq.txt:278]: Broke it into two sentences

**Pull request merged**